nsxt/provider.go

```diff
@@ -496,6 +496,7 @@ func Provider() *schema.Provider {
 			"nsxt_policy_gateway_flood_protection_profile_binding":     resourceNsxtPolicyGatewayFloodProtectionProfileBinding(),
 			"nsxt_policy_compute_sub_cluster":                          resourceNsxtPolicyComputeSubCluster(),
 			"nsxt_policy_tier0_inter_vrf_routing":                      resourceNsxtPolicyTier0InterVRFRouting(),
+			"nsxt_policy_vpc_security_policy":                          resourceNsxtPolicyVPCSecurityPolicy(),
```

Comment:
can we remove the first `policy` for all VPC resources? There is no other VPC.. Would love to hear also from @salv-orlando and @tvigneron

Reply:
I agree, also proposed that in design review.

Reply:
It seems a great idea - for vpc resources let's get rid of the Policy. We don't want the name to be unecessarily long